library/core/class.format.php

```diff
@@ -1939,7 +1948,7 @@ protected static function formatMentionsCallback($str) {
                 if (self::$DisplayNoFollow) {
                     $attributes['rel'] = 'nofollow';
                 }
-                $parts[$i] = anchor('@'.$mention, url(str_replace('{name}', rawurlencode($mention), self::$MentionsUrlFormat), true), 'atMention', $attributes).$suffix;
+                $parts[$i] = anchor('@'.$mention, url(str_replace('{name}', rawurlencode($mention), self::$MentionsUrlFormat), true), '', $attributes).$suffix;
```

Comment:
While we're in here, could expand this onto multiple lines. It's really difficult to follow what parameters is what with so much inline logic.

Example:
```php
anchor(
    '@' . $mention,
    url(str_replace(/***/, true),
    '',
    $attributes
) . $suffix;
```